lib/robots/dor_repo/gis_delivery/load_geoserver.rb

```diff
@@ -66,23 +62,24 @@ def layertype
                          end
         end
 
-        # @return [Hash] selectively parsed MODS record to match RGeoServer requirements
-        def layer
-          @layer ||= {
-            (raster? ? 'raster' : 'vector') => {
-              'druid' => bare_druid,
-              'title' => mods.full_titles.first,
-              'abstract' => mods.term_values(:abstract).compact.join("\n"),
-              'keywords' => [mods.term_values([:subject, 'topic']),
-                             mods.term_values([:subject, 'geographic'])].flatten.compact.collect(&:strip),
-              'format' => layertype
-            }
-          }
+        def title
+          @title ||= Cocina::Models::Builders::TitleBuilder.full_title(cocina_object.description.title).first
+        end
+
+        def abstract
+          @abstract ||= cocina_object.description.note.select { |note| note.type == 'abstract' || note.displayLabel&.downcase == 'abstract' }.map(&:value).join("\n")
```

Comment:
Is it important to call `.compact`  as well here?

Do we have abstracts in Cocina that has `note.displayLabel`  set to `abstract` without `note.type` being set to `abstract`?

Reply:
It is unlikely that there would be an abstract without a value, since I don't think there is ever a reason to have an abstract with a different type of value.

Yes, we have abstracts that have note.displayLabel set to abstract without note.type being set to abstract. The cocina object that was in the spec had one such abstract.

Reply:
Having one in the spec isn't having one in our actual data. This is another little Cocina wrinkle that would be nice to iron out. Maybe I'll keep a list ...

Reply:
The spec was created from actual data. See https://argo.stanford.edu/items/druid:dg548ft1892.json